WooCommerce/Classes/POS/Presentation/TotalsView.swift

```diff
@@ -5,6 +5,9 @@ struct TotalsView: View {
     @ObservedObject private var totalsViewModel: TotalsViewModel
     @ObservedObject private var cartViewModel: CartViewModel
 
+    /// Used for synchronizing totals fields animation
+    @Namespace private var totalsFieldAnimation
```

Comment:
True.

I'll pass matchedGeometryId through parameters, and use `UUID` for identifiers since they can have arbitrary values, most importantly they should be consistent and match between specific shimmering lines and text fields.

https://github.com/woocommerce/woocommerce-ios/pull/13437/commits/34ac5a2f908604b044e4bf8864f7466e878d29c0

Reply:
Good call, I will move the comment to the code. I also learned about it just now when doing the task and it was perfect for this case. https://github.com/woocommerce/woocommerce-ios/pull/13437/commits/7fd9c0be6c34a60ef071b514649ca865bb57c739